modules/viewing-scheduled-pipeline-runs.adoc

```diff
@@ -7,32 +7,29 @@
 You can view a list of pipeline runs that are scheduled for execution in {productname-short}. From this list, you can view details relating to your pipeline runs, such as the pipeline version that the run belongs to. You can also view the run status, execution frequency, and schedule.
 
 .Prerequisites
-
 * You have logged in to {productname-long}.
 ifndef::upstream[]
-* You have installed the OpenShift Pipelines operator.
 * If you are using specialized {productname-short} groups, you are part of the user group or admin group (for example, {oai-user-group} or {oai-admin-group} ) in OpenShift.
 endif::[]
 ifdef::upstream[]
-* You have installed the Data Science Pipelines operator.
 * If you are using specialized {productname-short} groups, you are part of the user group or admin group (for example, {odh-user-group} or {odh-admin-group}) in OpenShift.
 endif::[]
 * You have previously created a data science project that is available and contains a pipeline server.
 * You have imported a pipeline to an active and available pipeline server.
-* You have created and scheduled a pipeline run.
+* You have scheduled a pipeline run that is available to view.
 
 .Procedure
 . From the {productname-short} dashboard, click *Data Science Pipelines* -> *Runs*.
 +
 The *Runs* page opens.
 . From the *Project* list, select the project whose scheduled pipeline runs you want to view.
-. Click the *Scheduled* tab.
+. Click the *Schedules* tab.
 . Study the table showing a list of scheduled runs.
 +
-After a run has been scheduled, the run's status is displayed in the *Status* column in the table, indicating whether the run is ready for execution or unavailable for execution. To enable or disable a previously imported notebook image, on the row containing the relevant notebook image, click the toggle in the *Enabled* column.
+After a run has been scheduled, the run's status is displayed in the *Status* column in the table, indicating whether the run is ready for execution or unavailable for execution. To change its execution availability, on the row containing the run, click the toggle in the *Status* column.
```

Comment:
After a run has been scheduled, the run's status indicates whether the run is ready for execution or unavailable for execution. To change its execution availability,  click the run's  *Status* icon.